src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TeleporterSystem.java

```diff
@@ -57,8 +46,35 @@
     @ReceiveEvent(components = {SetTeamOnActivateComponent.class})
     public void onActivate(ActivateEvent event, EntityRef entity) {
         EntityRef player = event.getInstigator();
-        String team = setPlayerTeamToTeleporterTeam(player, entity);
-        handlePlayerTeleport(player, team);
+        if (properTeamSize(entity, player)) {
+            String team = setPlayerTeamToTeleporterTeam(player, entity);
+            handlePlayerTeleport(player, team);
+        }
+    }
+
+    private boolean properTeamSize(EntityRef teleporter, EntityRef player) {
+        int maxTeamSizeDifference = 3;
```

Comment:
Ideally, this should be a constant, even more ideally a "world-configurable" one as in when you start to host an LaS game, you can configure it and then it applies for everyone.
World configuration is currently being fixed though, so I'm not sure how well you can currently test that - but the general idea should be informed about in the world gen tutorial IIRC.

Reply:
@jdrueckert I don't remember seeing this there, but would it be ok if we could configure this through the settings, maybe under the player configuration part. If there is any better alternative could you point it to me, I will work on that.

Reply:
Did you mean configuring it through the universe setup. My game seems to crash when I try to configure LightAndShadow.

Reply:
Yes I meant configuring through the universe setup.
Huh, I'm also getting a `java.util.concurrent.ExecutionException: java.lang.ExceptionInInitializerError` :thinking: For `CoreGameplay` it works...
I recall there _was_ an issue at some point due to pre-generation stage in LaS trying to access something that's only loaded later (IIRC that came from `StaticCities`). However, I'm rather sure we fixed that :thinking: 

@ahv15 can you open up an issue for this?

Reply:
On second thought, the universe setup is only for world-gen stuff, so that's not really suitable for gameplay configuration :thinking:
The least we should do as a first step is create constants that allow for a central "configuration". As a follow up we can consider something like an in-game configuration mechanism for the host, such as a screen, some "buttons" or an additional dialogue option with the fool or something like that.

Reply:
Will do.

Reply:
An addition dialogue option might be preferable in this case as currently there aren't many configurable settings for LaS and adding a single screen/button seems unnecessary.

Reply:
While looking at some suitable alternatives I did see some issues, so for the time being I will just add a command through which we can set the maxTeamSizeDifference and once the list of configurations becomes larger maybe create a separate screen then.